goroutine 67 [running]:\nruntime/debug.Stack()\n\t/usr/local/go/src/runtime/debug/stack.go:26 +0x5e\ngithub.com/sourcegraph/conc/panics.NewRecovered(0x1, {0xb37c60, 0xc000238890})\n\t/home/eder/go/pkg/mod/github.com/sourcegraph/conc@v0.3.0/panics/panics.go:59 +0x85\ngithub.com/sourcegraph/conc/panics.(*Catcher).tryRecover(0xc000428090)\n\t/home/eder/go/pkg/mod/github.com/sourcegraph/conc@v0.3.0/panics/panics.go:28 +0x5f\npanic({0xb37c60?, 0xc000238890?})\n\t/usr/local/go/src/runtime/panic.go:787 +0x132\nfindit/internal/clients/googlemaps.(*Client).Reverse(0xc00011e1c0, {0xc0000f5136, 0x9}, {0xc0000f5140, 0x9})\n\t/home/eder/Workspace/findit-rio/internal/clients/googlemaps/client.go:55 +0x5c5\nfindit/internal/services.(*LocationStrategy).Execute(_, {{_, _}, {_, _}, {_, _}, {_, _}})\n\t/home/eder/Workspace/findit-rio/internal/services/geolocation_reverse.go:25 +0xe2\nfindit/internal/services.(*LocationFeeder).Feed.func1()\n\t/home/eder/Workspace/findit-rio/internal/services/location_feeder.go:34 +0xaf\ngithub.com/sourcegraph/conc/pool.(*ResultErrorPool[...]).Go.func1()\n\t/home/eder/go/pkg/mod/github.com/sourcegraph/conc@v0.3.0/pool/result_error_pool.go:27 +0x2c\ngithub.com/sourcegraph/conc/pool.(*ResultErrorPool[...]).Go.(*ErrorPool).Go.func2()\n\t/home/eder/go/pkg/mod/github.com/sourcegraph/conc@v0.3.0/pool/error_pool.go:30 +0x23\ngithub.com/sourcegraph/conc/pool.(*Pool).worker(0xc000343d28?)\n\t/home/eder/go/pkg/mod/github.com/sourcegraph/conc@v0.3.0/pool/pool.go:154 +0x69\ngithub.com/sourcegraph/conc/panics.(*Catcher).Try(0x0?, 0xc000142c60?)\n\t/home/eder/go/pkg/mod/github.com/sourcegraph/conc@v0.3.0/panics/panics.go:23 +0x42\ngithub.com/sourcegraph/conc.(*WaitGroup).Go.func1()\n\t/home/eder/go/pkg/mod/github.com/sourcegraph/conc@v0.3.0/waitgroup.go:32 +0x4d\ncreated by github.com/sourcegraph/conc.(*WaitGroup).Go in goroutine 8\n\t/home/eder/go/pkg/mod/github.com/sourcegraph/conc@v0.3.0/waitgroup.go:30 +0x73\n\n
